### PR TITLE
Pin chef and ohai versions

### PR DESCRIPTION
**Pull request by scottopherson**
Pinning chef and ohai versions so push-client can
build successfully.

### CODE REVIEW

File: omnibus/config/projects/push-jobs-client.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@
 conflict "opscode-push-jobs-client"
 
 build_iteration 1
-build_version "2.0.2"
+build_version "2.1.0"
```
Comment (schisamo) [inline omnibus/config/projects/push-jobs-client.rb]: How did we determine this version number?

Comment (scottopherson) [inline omnibus/config/projects/push-jobs-client.rb]: @ksubrama convinced me about this change but I'm hazy on the details so he's welcome to chime in. From what I remember a minor bump seemed appropriate given recent changes in omnibus-software.

**Comment by schisamo** [comment]
👍

**Comment by oferrigni** [comment]
:+1:

File: omnibus/config/projects/push-jobs-client.rb (hunk 1 above)
Comment (ksubrama) [inline omnibus/config/projects/push-jobs-client.rb]: I changed the ruby being shipped from 2.1.5 from rubyinstaller to 2.1.8 that we compile.  Also we don't compile devkit.  Now I'm pretty sure devkit was not part of the push jobs API but I figured it was safe to just bump this sucker and not worry about it...

**Pull request merged**